client/app/helpers/format-date.coffee

```diff
@@ -1,10 +1,11 @@
 `import Ember from 'ember'`
 
-formatDate = (date, options) ->
+formatDate = (date, format) ->
```

Comment:
This version doesn't work if the helper is not called with a second argument, that is, if it is called as `{{format-date currentDate}}`, because the second argument becomes an `options` argument.

With the old version, the `format=` named parameter was optional. You could call it as `{{format-date currentDate format="LL"}}` or as `{{format-date currentDate}}`.

Handlebar helper arguments seem really poorly documented. This is the best info I could find:

http://stackoverflow.com/questions/17954192/how-to-pass-parameters-to-handlebars-helper-whats-the-difference-between-optio